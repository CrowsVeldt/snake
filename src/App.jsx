import GameBoard from './gameBoard'
import GameLoop from './gameLoop'
import React from 'react'
import StartButton from './StartButton'

let App = () => {
  return (
    <div>
      <GameLoop />
      <GameBoard />
      <StartButton />
    </div>
  )
}

export default App
