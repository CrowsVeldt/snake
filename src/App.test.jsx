/* eslint-env jest */
import App from './App'
import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'

describe('The App Componenet', () => {
  const shallow = new ShallowRenderer()

  test('Renders without crashing', () => {
    const div = document.createElement('div')
    shallow.render(<App />, div)
  })

  test('Renders correctly', () => {
    const tree = shallow.render(<App />)
    expect(tree).toMatchSnapshot()
  })
})
