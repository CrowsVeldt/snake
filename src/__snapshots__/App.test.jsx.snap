// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App Componenet Renders correctly 1`] = `
<div>
  <Connect(GameLoop) />
  <Connect(gameLayout) />
  <Connect(Button) />
</div>
`;
